README.md

```diff
@@ -18,166 +18,4 @@ CloudFormation). If you try to use such a Lambda with an API Gateway
-
-* `wai-handler-hal`: the library that glues `wai` and `hal` together;
-* `wai-handler-hal-example`: A simple `servant` API set up to run on
-  both `hal` and `warp`; and
-* `wai-handler-hal-cdk`: A small [AWS
-  CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html)
-  application that deploys an API backed by `wai-handler-hal-example`.
-
-## Developing
-
-Instead of serving your application with something like
-[`warp`](https://hackage.haskell.org/package/warp), you will set up an
-executable that serves your application with
-[`hal`](https://hackage.haskell.org/package/hal). The necessary glue
-code looks something like this:
-
-```haskell
-import AWS.Lambda.Runtime (mRuntime)
-import Network.Wai (Application)
-import qualified Network.Wai.Handler.Hal as WaiHandler
-
-app :: Application
-app = undefined -- From Servant or wherever else
-
-main :: IO ()
-main = mRuntime $ WaiHandler.run app
-```
-
-### Local testing
-
-Compiled Lambda functions can be awkward to test, especially if you're
-relying on an API Gateway integration to translate HTTP
-requests. Consider defining a second executable target that serves
-your `Application` using `warp`, which you can use for local
-testing.
-
-### Caveats
-
-* The Lambda is never told the port that the API Gateway is listening
-  on. Most APIs will listen on `443` (HTTPS), so that's what's in this
-  example.
-
-* The Lambda is never told the HTTP version that the client uses when
-  talking with the API Gateway. We assume HTTP 1.1.
-
-## Packaging
-
-Lambda functions are packaged in one of two ways: as `.zip` files or
-as Docker container images. The `wai-handler-hal-cdk` example uses
-`.zip` files for simplicity. CDK doesn't give us an easy way to build
-container images using `nix build`, so a container-based deployment
-using CDK would need to push to ECR first and then reference the image
-by name.
-
-### `.zip` files
-
-You will need to create a `.zip` file containing ([at
-least](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-custom.html#runtimes-custom-build))
-an executable named `bootstrap`. This executable needs to run in
-Amazon's runtime environment, and there are multiple ways to ensure
-this.
-
-#### Static linking
-
-IOHK's [`haskell.nix`](https://github.com/input-output-hk/haskell.nix)
-can build static Haskell binaries by cross-compiling against musl
-libc. This is convenient, but consider copyleft implications (of
-`libgmp`; `wai-handler-hal` is BSD-3-Clause) if you are distributing
-the binaries to other people.
-
-#### Dynamic linking
-
-The other option is to compile the executable in an environment with
-packages that match the [Lambda runtime
-environment](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). Some
-ideas:
-
-* Build the executable in an "imitation environment" like the [`lambci/lambda`](https://hub.docker.com/r/lambci/lambda) Docker container; or
-* (Untested) build the executable on an Amazon Linux 2 EC2 instance.
```

Comment:
Much of this is still relevant for people learning to write Lambda Functions in Haskell — can we keep it and remove/clarify the references to the example projects?